src/pages/library/Code.js

```diff
@@ -26,9 +25,26 @@ const styles = {
   }
 }
 
-export const MethodCard = ({ method: { name, synopsis }, ...props }) => {
+export const MethodCard = ({ method, ...props }) => {
+  const isMethodsRepoMethod = method.namespace && method.name
+
+  let name
+  let href
+  let description
+
+  if (isMethodsRepoMethod) {
+    name = method.name
+    description = method.synopsis
+    href = `${getConfig().firecloudUrlRoot}/?return=${returnParam()}#methods/${method.namespace}/${method.name}/`
+  } else {
+    name = method.toolname
+    description = method.description
```

Comment:
Dockstore workflow descriptions contain Markdown, so it's less useful to display them here but the Featured Workspaces page also has unrendered Markdown descriptions.